src/annotator/integrations/test/pdf-metadata-test.js

```diff
@@ -202,135 +202,114 @@ describe('PDFMetadata', function () {
         fingerprint: 'fakeFingerprint',
       });
       const pdfMetadata = new PDFMetadata(fakePDFViewerApplication);
-      return pdfMetadata.getUri().then(function (uri) {
-        assert.equal(uri, 'http://fake.com/');
-      });
+      const uri = await pdfMetadata.getUri();
+      assert.equal(uri, 'http://fake.com/');
     });
   });
 
-  describe('metadata sources', function () {
-    let pdfMetadata;
+  const testMetadata = {
+    fingerprint: 'fakeFingerprint',
+    title: 'fakeTitle',
+    metadata: {
+      'dc:title': 'dcFakeTitle',
+    },
+    url: 'http://fake.com/',
+  };
+
+  function createPDFMetadata(metadata = testMetadata) {
     const fakePDFViewerApplication = new FakePDFViewerApplication();
     fakePDFViewerApplication.completeInit();
-    fakePDFViewerApplication.finishLoading({
-      fingerprint: 'fakeFingerprint',
-      title: 'fakeTitle',
-      metadata: {
-        'dc:title': 'dcFakeTitle',
-      },
-      url: 'http://fake.com/',
+    fakePDFViewerApplication.finishLoading(metadata);
+    return {
+      fakePDFViewerApplication,
+      pdfMetadata: new PDFMetadata(fakePDFViewerApplication),
+    };
+  }
+
+  describe('#getUri', () => {
+    it('returns the non-file URI', async () => {
```

Comment:
What exactly does _non-file_ mean here?

Reply:
It means any URL with a scheme other than `file:`. In practice it means a URL with a scheme that is `http:` or `https:`. I'll update the test description to clarify.